src/hmem_cuda.c

```diff
@@ -366,15 +364,6 @@ int cuda_get_base_addr(const void *ptr, void **base, size_t *size)
 	return -FI_EIO;
 }
 
-static cudaError_t cuda_disabled_cudaMemcpy(void *dst, const void *src,
```

Comment:
This change to core code, need to be in a separate PR

Reply:
@wzamazon do you want the change in lines 640 and 641 to be in line 746 in this PR and remove that code in a different PR? Or remove line 640 in this PR and remove the function definition in another PR?

Reply:
The change made to `src/hmem_cuda.c` should be in a separate PR.